apps/web/src/types/sanity.types.generated.ts

```
@@ -0,0 +1,1550 @@
+
+export type Grid = {
+	_type: 'grid';
+	title?: string;
+	columns: Columns;
+	items?: Array<
+		| {
+				asset?: {
+					_ref: string;
+					_type: 'reference';
+					_weak?: boolean;
+					[internalGroqTypeReferenceTo]?: 'sanity.imageAsset';
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)